tests/test_text_server.h

```diff
@@ -247,3 +249,4 @@ TEST_SUITE("[[TextServer]") {
 }; // namespace TestTextServer
 
 #endif // TEST_TEXT_SERVER_H
+#endif // TOOLS_ENABLED
```

Comment:
Add missing newline at end of file:

```suggestion
#endif // TOOLS_ENABLED
```

Reply:
OK I just git hooked this, should be fixed.